### PR TITLE
Updates for Lotand

### PR DESCRIPTION
**Pull request by jazg**
This pull request is for a few Ingress requirements in the internal Lotand tool. These include:

- Updating go-ethereum version as the version we were previously using is incompatible with the latest version of go.
- Exporting the Ingress config so it can be accessed externally.
- Migrating the list of approved traders to the config.

### CODE REVIEW

File: contract/config.go [hunk 1]
```diff
@@ -14,9 +19,16 @@ const (
 	NetworkLocal Network = "local"
 )
 
-// Config defines the different settings for connecting to Ethereum on
-// different Republic Protocol networks.
 type Config struct {
+	Ethereum                republicContract.Config `json:"republic"`
```
Comment (tok-kkk) [inline contract/config.go]: the json tag for this field should be `ethereum` or `config`

**Review by tok-kkk — CHANGES_REQUESTED** [review]

File: contract/config.go (hunk 1 above)
Comment (jazg) [reply contract/config.go]: This field was not modified from the [previous version](https://github.com/republicprotocol/renex-ingress-go/blob/42422c0941914ae4eb3f0385f1128d3482ea3f1d/cmd/ingress/ingress.go#L32). The only change I made was the `ApprovedTraders` field. Do you still want me to update it?

Comment (tok-kkk) [reply contract/config.go]: Instead, let's just change the go file to `Republic`

Comment (jazg) [reply contract/config.go]: Done

**Review by tok-kkk — APPROVED** [review]
Looks good

**Pull request merged**